src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+  const usedValues = [];
+
+  for (const key in robot) {
+    if (usedValues.includes(robot[key])) {
```

Comment:
You can check if this property exists in `inversedRobot` and avoid creating additional array.